MANUAL.md

```diff
@@ -128,7 +128,7 @@ These flags describe the sequences to use for adapter trimming in the RNA Seq da
 
     cutadapt:
         a: AGATCGGAAGAG  -> Adapter for trimming forward read
-        A: AGATCGGAAGAG  -> Adapter for trimming reverse read *FIXME: these really are the same sequence?*
+        A: AGATCGGAAGAG  -> Adapter for trimming reverse read *JUST CHECKING: these really are the same sequence?*
```

Comment:
yes they are the same for most Illumina libraries. If you have a custom library prep method, you can change this.